laravel-app/resources/views/post/show.blade.php

```diff
@@ -31,6 +31,9 @@
                         <div class="card">
                             <div class="card-body">
                                 <p>{{ $post->content }}</p>
+                                @if ($post->media)
+                                    <img src="{{ asset('storage/' . $post->media->file_path) }}" style="max-width: 100%; height: auto;" alt="Post Image">
```

Comment:
You could add 'storage/' on the paths attribute on the model's `getAttribute()` method so that you won't have to add it to every `asset()`. A little less tedious for you guys IMO.

Reply:
Yes sir will do 🙇